pkgs/development/python-modules/textual/default.nix

```diff
@@ -35,10 +41,9 @@ buildPythonPackage rec {
 
   dependencies =
     [
-      platformdirs
       markdown-it-py
+      platformdirs
       rich
-      typing-extensions
```

Comment:
┃        > Checking runtime dependencies for textual-1.0.0-py3-none-any.whl
┃        >   - typing-extensions not installed